dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java

```diff
@@ -1065,12 +1103,32 @@ protected boolean hasHref( List<String> fields )
     {
         if ( order != null && !StringUtils.isEmpty( order ) )
         {
-
             return Arrays.asList( order.split( "," ) );
         }
 
         return null;
     }
+    
+    private List<String> getGridOrderParams( String order, Map<String,String> dataElementOrders )
+    {
+        List<String> dataElementOrderList = new ArrayList<String>();
+        
+        if ( order != null && !StringUtils.isEmpty( order ) && dataElementOrders != null && dataElementOrders.size() > 0 )
```

Comment:
If you use StringUtils from Spring package, then you don't have to do null check yourself